gsrs-spring-legacy-indexer/src/main/java/ix/core/search/text/TextIndexer.java

```diff
@@ -3359,28 +3362,30 @@ private static String replaceSpecialCharsForExactMatch(String in) {
 	//TODO: this is a fairly hacky way to try to recreate simple character sequence-level
 	//functionality within lucene, and there needs to be a better way
 	private static String transformQueryForExactMatch(String in){
-
+        // This is called when doing searches and maybe other cases
 		String tmp =  START_PATTERN.matcher(in).replaceAll(TextIndexer.START_WORD);
 		tmp =  STOP_PATTERN.matcher(tmp).replaceAll(TextIndexer.STOP_WORD);
-		
-		
-		tmp =  LEVO_PATTERN.matcher(tmp).replaceAll(TextIndexer.LEVO_WORD);
-		tmp =  DEXTRO_PATTERN.matcher(tmp).replaceAll(TextIndexer.DEXTRO_WORD);
-        tmp =  RACEMIC_PATTERN.matcher(tmp).replaceAll(TextIndexer.RACEMIC_WORD);
-
-		return tmp;
+        for(StandardEncoding se: StandardEncodings.getEncodings()) {
+            tmp=se.encode(tmp);
+        }
+        return tmp;
 	}
 
 	private static final Pattern START_PATTERN = Pattern.compile(TextIndexer.GIVEN_START_WORD,Pattern.LITERAL );
 	private static final Pattern STOP_PATTERN = Pattern.compile(TextIndexer.GIVEN_STOP_WORD,Pattern.LITERAL );
 
-	private static final Pattern LEVO_PATTERN = Pattern.compile(Pattern.quote("(-)"));
```

Comment:
I don't know if these should stay here or go into the StandardEncodings  

If they are here, some needed to public.